packages/manager/modules/navbar/src/phone/template.html

```diff
@@ -0,0 +1,14 @@
+<a
+    data-ng-if="$ctrl.phoneNumber"
+    class="d-flex phone-link"
+    data-ng-href="{{ 'tel:' + $ctrl.phoneNumber }}"
+>
+    <span
+        class="align-self-center oui-icon oui-icon-phone"
+        aria-hidden="true"
+    ></span>
+    <span
+        class="align-self-center d-none d-md-inline phone-number"
+        data-ng-bind="$ctrl.phoneNumber"
+    ></span>
```

Comment:
No `sr-only` content to describe what the phone number is meant for ?

Reply:
This is done. Thanks!